bayesapi/resources/anomalies.py

```diff
@@ -12,12 +12,13 @@ def on_post(self, req, resp):
         target_column = req_vars['target-column']
         context_columns = req_vars['context-columns']
 
+        quoted_tgt_column = '"' + target_column + '"'
```

Comment:
We should be consistent in how we're doing string formatting... right below I'm using `.format()` to format the string with quotes around it, so either that should changes to the way you have it done here, or vice versa.